Fix typos and grammar in README files (#230)
apple/containerization: README.md

## Example Usage

> For examples of how to use some of the libraries' surface, the cctl executable is a good start. This tool's primary job is as a playground to try out the API. It contains commands that exercise some of the core functionality of the various products, such as:

Reviewer: Thanks for the PR! I think we should just go ahead and change this paragraph to say something like this instead

```suggestion
The cctl executable provides a good example of how to use the library's APIs. For example:
```

Reviewer: @cluster2600 Could you update to @dkovba's suggestion? Then I'm good to approve the PR. Thanks!

Author: @katiewasnothere Sure I have changed it.
